addons/web/static/tests/views/fields/many2many_tags_avatar_field_tests.js

```diff
@@ -361,14 +362,14 @@ QUnit.module("Fields", (hooks) => {
             target.querySelector(
                 ".o_kanban_record:nth-child(3) .o_field_many2many_tags_avatar img.o_m2m_avatar"
             ).dataset.src,
-            "/web/image/partner/5/avatar_128",
+            `${getOrigin()}/web/image/partner/5/avatar_128`,
             "should have correct avatar image"
         );
         assert.strictEqual(
             target.querySelectorAll(
                 ".o_kanban_record:nth-child(3) .o_field_many2many_tags_avatar img.o_m2m_avatar"
             )[1].dataset.src,
-            "/web/image/partner/4/avatar_128",
+            `${getOrigin()}/web/image/partner/4/avatar_128`,
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)